processes.go

```diff
@@ -6,12 +6,14 @@ import (
 	log "github.com/sirupsen/logrus"
 )
 
+//todo: make all JSON keys lowercase
```

Comment:
A reason why this is todo?

Reply:
I want to include this in the separate PR because to provide context
This is partly backward-incompatible change and will require some  changes from API side